### PR TITLE
[PREPARES FOR JENKINS-36871] Fix findbug errors and jelly files

### PR DESCRIPTION
**Pull request by stephenc**
@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/main/modules/instance_identity/PageDecoratorImpl.java [hunk 1]
```diff
@@ -23,6 +24,6 @@ public PageDecoratorImpl() throws IOException {
 
     public String getEncodedPublicKey() {
         RSAPublicKey key = identity.getPublic();
-        return new String(Base64.encodeBase64(key.getEncoded()));
+        return new String(Base64.encodeBase64(key.getEncoded()), Charset.forName("UTF-8"));
```
Comment (jglick) [inline src/main/java/org/jenkinsci/main/modules/instance_identity/PageDecoratorImpl.java]: `US-ASCII` no?

File: src/main/java/org/jenkinsci/main/modules/instance_identity/pem/PEMHelper.java [hunk 2]
```diff
@@ -184,7 +185,7 @@ private static RSAPrivateCrtKeySpec newRSAPrivateCrtKeySpec(byte[] keyInPkcs1) t
 
     private static void writeEncoded(byte[] bytes, BufferedWriter wr) throws IOException {
         char[] buf = new char[PEM_LINE_LENGTH];
-        bytes = DatatypeConverter.printBase64Binary(bytes).getBytes();
+        bytes = DatatypeConverter.printBase64Binary(bytes).getBytes("UTF-8");
```
Comment (jglick) [inline src/main/java/org/jenkinsci/main/modules/instance_identity/pem/PEMHelper.java]: Ditto, this is guaranteed to be ASCII.

File: src/main/java/org/jenkinsci/main/modules/instance_identity/pem/PEMHelper.java [hunk 3]
```diff
@@ -104,7 +105,7 @@ public static String encodePEM(@Nonnull KeyPair keys) throws IOException {
 
         try {
             ByteArrayOutputStream baos = new ByteArrayOutputStream();
-            BufferedWriter bw = new BufferedWriter(new PrintWriter(baos));
+            BufferedWriter bw = new BufferedWriter(new PrintWriter(new OutputStreamWriter(baos, "UTF-8")));
```
Comment (jglick) [inline src/main/java/org/jenkinsci/main/modules/instance_identity/pem/PEMHelper.java]: Presumably ASCII.

File: src/main/java/org/jenkinsci/main/modules/instance_identity/PageDecoratorImpl.java (hunk 1 above)
Comment (stephenc) [inline src/main/java/org/jenkinsci/main/modules/instance_identity/PageDecoratorImpl.java]: Which is a subset of UTF-8

File: src/main/java/org/jenkinsci/main/modules/instance_identity/pem/PEMHelper.java (hunk 3 above)
Comment (stephenc) [inline src/main/java/org/jenkinsci/main/modules/instance_identity/pem/PEMHelper.java]: A subset of UTF-8

File: src/main/java/org/jenkinsci/main/modules/instance_identity/pem/PEMHelper.java (hunk 2 above)
Comment (stephenc) [inline src/main/java/org/jenkinsci/main/modules/instance_identity/pem/PEMHelper.java]: Which is guaranteed to also be UTF-8

**Pull request merged**